sdks/python/apache_beam/runners/common.py

```diff
@@ -188,3 +188,35 @@ def counter_for(self, aggregator):
     """Looks up the counter for this aggregator, creating one if necessary."""
     return self._counter_factory.get_aggregator_counter(
         self.step_name, aggregator)
+
+
+class DoFnContext(object):
```

Comment:
We won't have the two for a while, but we need to declare this first so we can start using it from the worker code.